README.md

```diff
@@ -73,11 +81,12 @@ This is an **external** notebook included in the [MintPy](https://github.com/ins
 Please cite the original reference as provided in the [ARIA-tools repo](https://github.com/aria-tools/ARIA-tools).
 
 ------
-## Contributors    
+## Contributors
 * David Bekaert
 * Simran Sangha
 * Emre Havazli
 * Brett Buzzanga
+* Robert Zinke
```

Comment:
@rzinke please also issue a PR for the ARIA-tools page to add yourself as contributor!
Thanks for all the contributions to ARIA-tools.